R/cv_measures.R

```diff
@@ -0,0 +1,86 @@
+#' Calculate Coefficient of Variation subtypes
+#'
+#' @description
+#' The function cv_measures produces CV subtype values in a tibble object.
+#'
+#' @usage
+#' cv_measures(data, dt0 = NULL, inter_gap = 45, tz = "" )
+#'
+#' @inheritParams CGMS2DayByDay
+#' @inheritParams conga
+#'
+#'
+#' @return When a data.frame object is passed, then a tibble object with
+#' three columns: subject id and corresponding CV subtype values is returned.
+#'
+#'
+#' @export
+#'
+#' @details
+#'
+#' A tibble object with 1 row for each subject, a column for subject id and
+#' a column for each cv subtype values is returned.
+#'
+#' Missing values will be linearly interpolated when close enough to non-missing values.
+#'
+#' \enumerate{
+#' \item Mean:
```

Comment:
Great job on the documentation! However, I suggest changing the names of returned parameters to avoid confusion with regular mean and sd. See examples for sd_measures names. Perhaps something like CVmean and CVsd?